app/src/lib/stores/app-store.ts

```diff
@@ -1470,6 +1471,31 @@ export class AppStore extends TypedBaseStore<IAppState> {
       return false
     }
 
+    // were any of these conflicted
+    const inAConflictedMerge: boolean = [...status.workingDirectory.files].some(
+      file => file.status === AppFileStatus.Resolved || file.status === AppFileStatus.Conflicted
+    )
+    const prevouslyInAConflictedMerge = false
```

Comment:
The start of our wishful thinking. We've encoded what we'd like to inspect in the comments